### PR TITLE
gcloud: add German translation

### PR DESCRIPTION
**Pull request by jnmcfly**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/gcloud.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# gcloud
+
+> Das offizielle CLI-Tool für die Google Cloud Platform.
+> Weitere Informationen: https://cloud.google.com/sdk/gcloud.
+
+- Alle Eigenschaften in deiner aktiven Konfiguration auflisten:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:6]:
```suggestion
- Liste alle Eigenschaften der aktiven Konfiguration auf:
```

File: pages.de/common/gcloud.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud config list`
+
+- Bei deinem Google-Konto anmelden:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:10]:
```suggestion
- Mit einem Google-Konto anmelden:
```

Imperative is very hard in this case, I wouldn't know how to do it.

File: pages.de/common/gcloud.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud auth login`
+
+- Das aktive Projekt festlegen:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:14]:
```suggestion
- Lege das aktive Projekt fest:
```

File: pages.de/common/gcloud.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud config set project {{Projektname}}`
+
+- Eine SSH-Verbindung zu einer virtuellen Maschineninstanz herstellen:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:18]:
```suggestion
- Stelle eine SSH-Verbindung zu einer virtuellen Maschineninstanz her:
```

File: pages.de/common/gcloud.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud compute ssh {{Benutzer}}@{{Instanz}}`
+
+- Alle Google Compute Engine-Instanzen in einem Projekt anzeigen. Instanzen aus allen Zonen werden standardmäßig aufgelistet:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:22]:
```suggestion
- Zeige alle Google Compute Engine-Instanzen in einem Projekt an. Standardmäßig werden Instanzen aus allen Zonen aufgelistet:
```

File: pages.de/common/gcloud.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud compute instances list`
+
+- Eine kubeconfig-Datei mit den entsprechenden Anmeldeinformationen aktualisieren, um kubectl auf einen bestimmten Cluster in Google Kubernetes Engine auszurichten:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:26]:
```suggestion
- Aktualisiere eine kubeconfig-Datei mit den entsprechenden Anmeldeinformationen, um kubectl auf einen bestimmten Cluster in Google Kubernetes Engine auszurichten:
```

File: pages.de/common/gcloud.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud container clusters get-credentials {{Clustername}}`
+
+- Alle gcloud CLI-Komponenten aktualisieren:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:30]:
```suggestion
- Aktualisiere alle gcloud CLI-Komponenten:
```

File: pages.de/common/gcloud.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud components update`
+
+- Hilfe für einen bestimmten Befehl anzeigen:
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:34]:
```suggestion
- Zeige Hilfe für einen bestimmten Befehl an:
```

File: pages.de/common/gcloud.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+`gcloud help {{Befehl}}`
+
```
Comment (gutjuri) [inline pages.de/common/gcloud.md:37]:
```suggestion
```

Comment (gutjuri) [reply pages.de/common/gcloud.md:37]: Remove empty line

**Review by gutjuri — CHANGES_REQUESTED** [review]
Thanks for your contribution. In general, we use imperative in command descriptions. Apart from that, it looks good.

**Review by gutjuri — APPROVED** [review]
Thank you!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**